### PR TITLE
PR - Introduce new API version (2019-01-01) with new (Export-related APIs) introduced

### PR DESCRIPTION
**Pull request by shbha**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by praries880** [comment]
@shbha Is the new stable version based directly on a preview version (like [2018-12-01](https://github.com/Azure/azure-rest-api-specs/blob/02de3fd5990ca41f34d87cf725ebc4b0a6741697/specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-12-01-preview/costmanagement.json) for example)?

Also have these changes been through an Azure API review?

**Comment by shbha** [comment]
The changes here are related to PR - https://github.com/Azure/azure-rest-api-specs/pull/5020 which has already been signed off by ARM.

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json [hunk 1]
```diff
@@ -0,0 +1,3417 @@
...
+          "nextLinkName": null
+        }
+      }
+    },
+	"/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/Microsoft.CostManagement/Query": {
+      "post": {
+        "tags": [
+          "Query"
+        ],
+        "operationId": "Query_UsageByManagmentGroup",
```
Comment (nschonni) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json]: Managment -> Management

**Comment by shbha** [comment]
@praries880 - would you be able to assist with the failing build? It started to fail after the readme files were updated. Am I not supported to update these files?

**Comment by praries880** [comment]
@shbha  There is a large delta (~1300 lines) between the [swagger in the stable version](https://github.com/Azure/azure-rest-api-specs/blob/1fd359a4268074a5afb5b0706411b500260ea8ac/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json) and the one you got reviewed by ARM for the [preview version](https://github.com/Azure/azure-rest-api-specs/blob/261c28346d702b3c316482a2467893d99c1e3183/specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-01-01-preview/costmanagement.json). I will wait for ARM to review this. 
Thanks for understanding :)

**Comment by ravbhatnagar** [comment]
As discussed over IM, can you abstract out the scope into a param called scope? That way you will reduce the number of APis you have to define. Right now each API is defined for every supported scopes (5 scopes I think). and scope can be an enum with a defined set of values. This will simplify generated SDKs and allow you to easily expand to new scopes in future if needed. 
@sanjaiganesh - other than this, it looks good.

**Comment by shbha** [comment]
> As discussed over IM, can you abstract out the scope into a param called scope? That way you will reduce the number of APis you have to define. Right now each API is defined for every supported scopes (5 scopes I think). and scope can be an enum with a defined set of values. This will simplify generated SDKs and allow you to easily expand to new scopes in future if needed.
> @sanjaiganesh - other than this, it looks good.

@sanjaiganesh @ravbhatnagar  - The recommended changes have been incorporated. All API paths are now using scope.

File: specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json [hunk 2]
```diff
@@ -0,0 +1,1438 @@
...
+          }
+        },
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope at which to list dimensions. This includes '/subscriptions/{subscriptionId}/' for subscription scope, '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for resourceGroup scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}' for Department scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope and '/providers/Microsoft.Management/managementGroups/{managementGroupId}' for Management Group scope.",
+            "x-ms-skip-url-encoding": true
```
Comment (sanjaiganesh) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-01-01/costmanagement.json]: use $ref approach and define it once to use at all other places.. 
{
            "$ref": "#/parameters/scopeParameter"
          },

**Review by sanjaiganesh — APPROVED** [review]
:shipit:

**Review by praries880 — APPROVED** [review]

**Pull request merged**